code-challenges.js

```diff
@@ -29,17 +29,44 @@ const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
 // Pseudo code:
+// input: two arrays 
+// expected output: length/a number
+// concantenate two arrays and then use .length to find the expected out put
+
+
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length)
+
+
+
 
 // --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
 
 const currentCohort = "Golf 2023"
 // Expected output: "3202 floG"
 
-// Pseudo code:
+// Pseudo code: 
+// input is a string; output should be the string in reversed order
+// convert the string to an array and then reverse the array, then print the array 
+// console.log(currentCohort.split().reverse()) --> output 9
+// [ 'Golf 2023' ]
+// console.log(currentCohort.split("")
+// console.log(currentCohort.split("").reverse())
+//both inpout and output should be strings, so the workflow should be like string-->convert to array-->reverse-->convert tostring
```

Comment:
I would leave code out of the pseudo code as much as you can. Here is how you could rewrite this:
- convert string to array using split
- reverse array using reverse
- convert array back to string using join

Reply:
gotcha, thanks Chris!